verification/VerifyCheckFingerprint.wdl

```diff
@@ -0,0 +1,51 @@
+version 1.0
+
+import "../verification/VerifyMetrics.wdl" as MetricsVerification
+import "../verification/VerifyTasks.wdl" as Tasks
+
+## Copyright Broad Institute, 2021
+##
+## This WDL script is designed to verify (compare) the outputs of the CheckFingerprint wdl.
+##
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+workflow VerifyCheckFingerprint {
+
+  input {
+    Array[File] test_metrics
```

Comment:
Does this work when these files are not defined? I.e. since these are optional pipeline outputs, should they be optional validation inputs?

Reply:
Yes, I think I have this working.